e2e-tests/src/cases.rs

```diff
@@ -6,8 +6,9 @@ use crate::{
         era_payouts_calculated_correctly as test_era_payout, era_validators as test_era_validators,
         fee_calculation as test_fee_calculation, finalization as test_finalization,
         staking_era_payouts as test_staking_era_payouts,
-        staking_new_validator as test_staking_new_validator, token_transfer as test_token_transfer,
-        treasury_access as test_treasury_access, validators_rotate as test_validators_rotate,
+        staking_new_validator as test_staking_new_validator, test_disable_node,
```

Comment:
Nit: would probably keep the re-export convention.

Reply:
Renamed.